src/Scoringline/SendinblueApi/Api/Email.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+/**
+ * This file is a part of scoringline sendinblue api package
+ *
+ * (c) Scoringline <m.veber@scoringline.com>
+ *
+ * For the full license, take a look to the LICENSE file
+ * on the root directory of this project
+ */
+namespace Scoringline\SendinblueApi\Api;
+
+use GuzzleHttp\Exception\RequestException;
+use Nekland\BaseApi\Api\AbstractApi;
+use Scoringline\SendinblueApi\Exception\EmailSendFailureException;
+use Scoringline\SendinblueApi\Model\Email as EmailModel;
+
+/**
+ * Class Email
+ * @package Scoringline\SendinblueApi\Api
+ * @author Joni Rajput <joni@sendinblue.com>
+ */
+class Email extends AbstractApi
+{
+    const API_URL = '/email';
+
+    /**
+     * @param array $from
+     * @param array $to
+     * @param string $subject
+     * @param string $content
+     * @param array $attachment Ie: ['YourFileName.Extension' => 'Base64EncodedChunkData'] to send attachment/s generated on the fly
+     * @param array $extraParams Ie: cc, bcc, replyTo, headers, text
+     * @return array
+     * @throws EmailSendFailureException
+     */
+    public function sendSimpleEmail($from, $to, $subject, $content, $attachment = [], $extraParams = [])
+    {
+        try {
+            return $this->post(self::API_URL, json_encode(array_merge([
+                'to' => $to,
+                'from' => $from,
+                'subject' => $subject,
+                'html' => $content,
+                'attachment' => $attachment
+            ], $extraParams)));
+        } catch(RequestException $e) {
+           throw new EmailSendFailureException($e->getResponse()->getBody());
+        }
+    }
+
+    /**
+     * @param EmailModel $emailModel
+     * @return array
+     * @throws EmailSendFailureException
+     */
+    public function sendEmail(EmailModel $emailModel)
+    {
+        try {
+            return $this->post(self::API_URL, json_encode($this->getParameters($emailModel)));
```

Comment:
re-using the method `sendEmailWithData` would avoid the try catch repetition ;) (same notice for `sendSimpleEmail`)